### PR TITLE
Feedback from Proteomics Lab working group

### PR DESCRIPTION
**Pull request by supernord**

### CODE REVIEW

**Comment by supernord** [comment]
@neoformit 😄

File: webapp/home/subdomains/proteomics/content/database_searching.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                'label': 'Input FASTA file(s), each containing a database',
+            }
+        ],
+        "button_link": "https://proteomics.usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/openms_decoydatabase/DecoyDatabase/",
+    },
+    {
+        "title_html": '<code>MaxQuant</code>',
+        "description_html": '<p>MaxQuant is a quantitative proteomics software package designed for analyzing large mass-spectrometric data sets.</p>',
+        "inputs": [
+            {'datatypes': ['thermo.raw']},
```
Comment (neoformit) [inline webapp/home/subdomains/proteomics/content/database_searching.py:45]: These first three look like they should be one input with three accepted datatypes, not three separate inputs? This implies that MaxQuant requires a `thermo.raw`, `mzMl` AND `mzXML` input to work! And maybe add a label?

**Review by neoformit — CHANGES_REQUESTED** [review]
One comment to address Johan, otherwise this looks good

**Comment by neoformit** [comment]
Hey @supernord are you running this locally, I can't remember?

**Comment by supernord** [comment]
hey @neoformit - I think I've fixed the issue with the datatypes
I'm not running this locally at the moment

**Comment by neoformit** [comment]
> I'm not running this locally at the moment
Let me know if you want to - it can be useful to preview changes as you make them

**Comment by supernord** [comment]
Thanks @neoformit 
What is required to run the media site locally?

**Comment by neoformit** [comment]
https://github.com/usegalaxy-au/galaxy-media-site/wiki/Development#developer-setup 

Short answer - 
```sh
git clone
virtualenv venv
pip install -r requirements.txt
``
`

**Review by neoformit — CHANGES_REQUESTED** [review]

File: webapp/home/subdomains/proteomics/content/database_searching.py [hunk 2]
```diff
@@ -42,9 +42,10 @@
         "title_html": '<code>MaxQuant</code>',
         "description_html": '<p>MaxQuant is a quantitative proteomics software package designed for analyzing large mass-spectrometric data sets.</p>',
         "inputs": [
-            {'datatypes': ['thermo.raw']},
-            {'datatypes': ['mzML']},
-            {'datatypes': ['mzXML']},
+            {
+                'datatypes': ['thermo.raw, mzML, mzXML'],
+                'label': 'Thermo raw, mzML, or mzXML files'
```
[the remark this answers is not in the document]
Comment (supernord) [reply webapp/home/subdomains/proteomics/content/database_searching.py]: Thanks @neoformit - updated to make this clear

File: webapp/home/subdomains/proteomics/content/database_searching.py [hunk 3]
```diff
@@ -44,7 +44,7 @@
         "inputs": [
             {
                 'datatypes': ['thermo.raw, mzML, mzXML'],
-                'label': 'Thermo raw, mzML, or mzXML files'
+                'label': 'Mass spectrometry data sets: Thermo raw, mzML, or mzXML files'
```
Comment (neoformit) [inline webapp/home/subdomains/proteomics/content/database_searching.py:47]: Sorry, can you delete `: Thermo raw, mzML, or mzXML files` as this is just a repeat of the `datatype` information

**Review by neoformit — CHANGES_REQUESTED** [review]

File: webapp/home/subdomains/proteomics/content/conversion_modification.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    #     "view_link": "",
+    #     "view_html": "",
+    #     "view_tip": "",
+    # },
+    {
+        "title_html": '<code>msconvert</code>',
+        "description_html": '<p>Convert and/or filter mass spectrometry files.</p>',
+        "inputs": [
+            {
+                'datatypes': ['thermo.raw', 'mzML', 'mzXML', 'raw', 'wiff', 'wiff.tar', 'agilentbrukeryep.d.tar', 'agilentmasshunter.d.tar', 'brukerbaf.d.tar', 'brukertdf.d.tar', 'watersmasslynx.raw.tar'],
```
Comment (neoformit) [inline webapp/home/subdomains/proteomics/content/conversion_modification.py:35]: This is how it should be done

File: webapp/home/subdomains/proteomics/content/database_searching.py [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+            },
+        ],
+        "button_link": "https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/maxquant/maxquant/",
+    },
+    {
+        "title_html": '<code>Morpheus</code>',
+        "description_html": '<p>Database search algorithm for high-resolution tandem mass spectra.</p>',
+        "inputs": [
+            {
+                'datatypes': ['mzml'],
```
Comment (neoformit) [inline webapp/home/subdomains/proteomics/content/database_searching.py:61]: This should be `mzMl` to match other instances

File: webapp/home/subdomains/proteomics/content/database_searching.py [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+                'label': 'Indexed mzML',
+            },
+            {
+                'datatypes': ['fasta,uniprotxml'],
```
Comment (neoformit) [inline webapp/home/subdomains/proteomics/content/database_searching.py:65]: `['fasta,uniprotxml']` should be two strings, not a string with internal comma

File: webapp/home/subdomains/proteomics/content/database_searching.py [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+            },
+        ],
+        "button_link": "https://proteomics.usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/morpheus/morpheus/",
+    },
+    {
+        "title_html": '<code>MSFraggerAdapter</code>',
+        "description_html": '<p>Peptide Identification with MSFragger.</p>',
+        "inputs": [
+            {
+                'datatypes': ['mzml,mzxml'],
```
Comment (neoformit) [inline webapp/home/subdomains/proteomics/content/database_searching.py:76]: As above on casing (`mzMl`, `mzXML`) and internal comma

**Review by neoformit — CHANGES_REQUESTED** [review]
Sorry just spotted a bunch of other stuff

**Comment by supernord** [comment]
Done 😄 
@neoformit

**Review by neoformit — APPROVED** [review]

**Comment by neoformit** [comment]
Hey @supernord could you please merge against `usegalaxy-au:main`?

**Pull request merged**